rllib/examples/curriculum/curriculum_learning.py

```diff
@@ -148,6 +148,15 @@ def _remote_fn(env_runner, new_task: int):
 class EnvTaskCallback(RLlibCallback):
     """Custom callback implementing `on_train_result()` for changing the envs' maps."""
 
+    def on_algorithm_init(
```

Comment:
nit: In this file, we have the `RLlibCallback` class inside the example script, but for the Atari one we keep it in a separate file. Should be also move the atari callback into the Atari example script?
I know we don't have a consistent rule for these custom classes as to where to put them yet. Just a thought, feel free to ignore for now.